src/components/PermissionsAccordion/components/PermissionsModal/PermissionsModal.js

```diff
@@ -36,8 +36,8 @@ class PermissionsModal extends React.Component {
           excludePermissionSets,
         } = props;
         const query = [
-          ...(listInvisiblePerms || ['visible==true']),
-          ...(excludePermissionSets && ['mutable==false']),
+          ...(listInvisiblePerms ? [] : ['visible==true']),
+          ...(excludePermissionSets ? ['mutable==false'] : []),
```

Comment:
@OleksandrAntonenko1, you wrote the original code here; does this change look correct to you? The original intent was a bit hard to decipher but I believe this preserves the logic.